app/policies/user_policy.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class UserPolicy < ApplicationPolicy
+  def new?
+    admin_only
+  end
+
```

Comment:
I don't think we want to touch users in this PR :)

Reply:
Yeah, think this was my misunderstanding with the authorise User in the new action